src/app/zap-templates/zcl/data-model/chip/content-launch-cluster.xml

```diff
@@ -30,83 +30,84 @@ limitations under the License.
 
     <command source="client" code="0x00" name="LaunchContent" response="LaunchResponse" optional="true">
       <description>Upon receipt, this SHALL launch the specified content with optional search criteria.</description>
-      <arg name="search" type="ContentSearch"/>
-      <arg name="autoPlay" type="BOOLEAN"/>
-      <arg name="data" type="CHAR_STRING" optional="true"/>
+      <arg name="Search" type="ContentSearchStruct"/>
+      <arg name="AutoPlay" type="BOOLEAN"/>
+      <arg name="Data" type="CHAR_STRING" optional="true"/>
```

Comment:
spec seems to say octet string

Reply:
Hmm.  That will break on-the-wire interop.  Let me think about this a bit.

Reply:
Filed https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/6204